app/helpers/application_helper.rb

```diff
@@ -528,6 +528,57 @@ def will_paginate(collection_or_options = nil, options = {})
     super(*[collection_or_options, options].compact)
   end
 
+  include Pagy::Frontend
+  # Cf https://github.com/ddnexus/pagy/blob/master/gem/lib/pagy/frontend.rb
+  def pagy_nav(pagy, id: nil, aria_label: nil, **vars)
+    return nil unless pagy
+
+    # Keep will_paginate behavior of showing nothing if only one page
+    return nil if pagy.series.length <= 1
+
+    id = %( id="#{id}") if id
+    a  = pagy_anchor(pagy, **vars)
+
+    html = %(<h4 class="landmark heading">#{t('pagination.title')}</h4>)
+
+    html << %(<ol#{id} class="pagination actions pagy" role="navigation" aria-label="#{aria_label || t('pagination.aria-label')}">)
+
+    prev_text = t("pagination.previous")
```

Comment:
Is it possible to make `pagy_t('pagy.prev')` output what we want by defining `pagy.prev` in our own view locales file?

Reply:
With an extra line, yes: https://ddnexus.github.io/pagy/docs/extras/i18n/ Code updated